codegen/go.gsl

```diff
@@ -282,6 +282,7 @@ function go_derive_dependency_type (dependency)
     check_arguments (my, "dependency", my)
 
     my.project_uid = component_derive_project_uid (my.dependency)
+    my.is_pointer = "0"
 
     if defined (my.dependency.interface)
         my.type_uid = make_id ("interface", "$(my.dependency.interface:Pascal)")
```

Comment:
What about the interface?
IMHO also should be a pointer.

Reply:
Only struct should be set as a pointer.